### PR TITLE
batsignal: update to 1.5.0.

### PR DESCRIPTION
**Pull request by motorto**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64{glibc,musl}

https://github.com/electrickite/batsignal/issues/21

### CODE REVIEW

File: srcpkgs/batsignal/patches/makefile.patch [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+--- a/Makefile
++++ b/Makefile
+@@ -3,47 +3,33 @@
+ NAME = batsignal
+ VERSION != grep VERSION version.h | cut -d \" -f2
+ 
+-CC = cc
+-RM = rm -f
+-INSTALL = install
+-SED = sed
```
Comment (sgn) [inline srcpkgs/batsignal/patches/makefile.patch:21]: Don't patch those variables out. They're noise.

File: srcpkgs/batsignal/patches/makefile.patch [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
++	sed "s/VERSION/$(VERSION)/g" < $(NAME).1.in > $@
+ 
+ install: all
+ 	@echo Installing in $(DESTDIR)$(PREFIX)
+-	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
+-	$(INSTALL) -d $(DESTDIR)$(MANPREFIX)/man1
+-	$(INSTALL) -m 0755 $(NAME) $(DESTDIR)$(PREFIX)/bin/
+-	$(INSTALL) -m 0644 $(NAME).1 $(DESTDIR)$(MANPREFIX)/man1/
++	install -d $(DESTDIR)$(PREFIX)/bin
++	install -d $(DESTDIR)$(MANPREFIX)/man1
```
Comment (sgn) [inline srcpkgs/batsignal/patches/makefile.patch:61]: And those lines, too

File: srcpkgs/batsignal/patches/makefile.patch [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ 
+ install-service: install
+-	$(INSTALL) -d $(DESTDIR)$(PREFIX)/lib/systemd/user
+-	$(INSTALL) -m 0644 $(NAME).service $(DESTDIR)$(PREFIX)/lib/systemd/user/
++	install -d $(DESTDIR)$(PREFIX)/lib/systemd/user
++	install -m 0644 $(NAME).service $(DESTDIR)$(PREFIX)/lib/systemd/user/
+ 
+ uninstall:
+ 	@echo Removing files from $(DESTDIR)$(PREFIX)
+--
```
Comment (sgn) [inline srcpkgs/batsignal/patches/makefile.patch:73]: And those lines.

**Comment by motorto** [comment]
@sgn Is that what you meant ? (Pushed changes to makefile) I didn't understand what you meant. (It still doesn't compile this way) gives me:

```text
=> batsignal-1.5.0_1: running do_build ...
cc -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe    -fdebug-prefix-map=/builddir/batsignal-1.5.0=.      -c -o batsignal.o batsignal.c
sed "s/VERSION/1.5.0/g" < batsignal.1.in > batsignal.1
In file included from batsignal.c:22:
/usr/include/libnotify/notify.h:25:10: fatal error: glib.h: No such file or directory
   25 | #include <glib.h>
      |          ^~~~~~~~
compilation terminated.
make: *** [<builtin>: batsignal.o] Error 1
=> ERROR: batsignal-1.5.0_1: do_build: '${make_cmd} CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" CPP="$CPP" AS="$AS" OBJCOPY="$OBJCOPY" OBJDUMP="$OBJDUMP" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-makefile.sh:9
```

Which makes me think that I am overriding still the flags

**Comment by motorto** [comment]
Thanks @sgn was failing to pass the $libs it seems, thanks for the huge help !

**Pull request merged**